packages/widget-chat/src/root.js

```diff
@@ -1,13 +1,19 @@
 import React, {PropTypes} from 'react';
 import {Provider} from 'react-redux';
-
+import {addLocaleData, IntlProvider} from 'react-intl';
+import enLocaleData from 'react-intl/locale-data/en';
+import messages from './locales/en';
 import ChatWidget from './containers/chat-widget';
 import store from './store';
 
 export default function Root({accessToken, userId}) {
+  addLocaleData(enLocaleData);
+
   return (
     <Provider store={store}>
-      <ChatWidget accessToken={accessToken} userId={userId} />
+      <IntlProvider locale={`en`} messages={messages}>
```

Comment:
We will when we actually implement the different languages in another user story.